src/components/header/header.js

```diff
@@ -0,0 +1,44 @@
+import { useRef } from "react";
+
+import PropTypes from "prop-types";
+
+import * as styledHeader from "./header.module.scss";
+
+const Header = props => {
+  const headerRef = useRef(null);
```

Comment:
Why do you use this?

Reply:
The useRef stores a value throughout the life of the component, so, I guess the useRef in this case is storing the header value for use on all pages. But, I don't know how the value is changed to the headerRef, since it doesn't redefine it. So, is it always null? I will delete it and see what happens